### PR TITLE
Develop ed quantities

### PR DESCRIPTION
**Pull request by evan1024**

### CODE REVIEW

**Comment by evan1024** [comment]
What do you think would be a good name for getting the skuLocationQuantity
that points to the same location and sku that the stock does? Relative?
Correspondent?

**Comment by gerickson808** [comment]
Corresponding would make sense

File: model/entity/Stock.cfc [hunk 1]
```diff
@@ -119,6 +119,10 @@ component displayname="Stock" entityname="SlatwallStock" table="SwStock" persist
 		return representation;
 	}
 	
+	public any function getEquivalentSkuLocationQuantity(){
```
[the remark this answers is not in the document]
Comment (sumitverma) [reply model/entity/Stock.cfc]: Isn't this just stock quantity? Stock is sku at a location.

[the remark this answers is not in the document]
Comment (evan1024) [reply model/entity/Stock.cfc]: Those two entities' calculated properties were getting out of sync, so we use one to get the other and make sure they match.

**Comment by evan1024** [comment]
Fixed the name.

**Pull request merged**